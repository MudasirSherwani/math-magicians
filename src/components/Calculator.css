
* {
  box-sizing: border-box;
  margin: 0;
  padding: 0;
}

.calculator-page {
  display: flex;
  justify-content: left;
  padding: 22px;
}

.heading > h2 {
  margin-top: 40px;
  font-size: 36px;
  text-align: center;
}

.calculator {
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: flex-end;
  margin-top: 5%;
  margin-left: 22vw;
  border: 2px solid black;
}

.display-value {
  width: 402px;
  height: 60px;
  display: flex;
  flex-direction: row;
  justify-content: flex-end;
  align-items: center;
  background-color: #858694;
  color: #fff;
  padding-inline-end: 5px;
  text-align: right;
  padding: 17px;
  font-size: 30px;
}

.inputs {
  display: grid;
  flex-direction: column;
  justify-content: center;
  grid-template-columns: auto auto auto auto;
}

button {
  width: 100px;
  height: 70px;
  margin: 0.5px;
  border: none;
  background-color: #e1e2e9;
}

button:hover {
  background-color: #d4d5da;
}

.display-value::placeholder {
  color: #fff;
}

.btn-org {
  background-color: #f5913e;
}

.btn-org:hover {
  background-color: #f48020;
}

.btn-zero {
  grid-column: 1 / 3;
  width: 200px;
}
