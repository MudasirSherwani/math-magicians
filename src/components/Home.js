import React from 'react';
import './home.css';

const Home = () => (
  <div className="homepage">
    <h2 className="heading">Welcome to our page!</h2>
    <p className="details">
      Lorem ipsum dolor sit amet consectetur adipisicing elit.
      Fugit ex fugiat voluptatibus accusamus possimus perspiciatis magni non similique odio debitis,
      deleniti reprehenderit quae beatae nulla nobis at explicabo dignissimos
      commodi mollitia provident quos cupiditate aspernatur.
      Sit quibusdam, porro quis atque voluptatem esse sed explicabo debitis.
      Earum reprehenderit non sit a repellat incidunt nemo odit vel.
    </p>
    <p className="details">
      Lorem ipsum dolor sit amet consectetur adipisicing elit.
      Fugit ex fugiat voluptatibus accusamus possimus perspiciatis magni non similique odio debitis,
      deleniti reprehenderit quae beatae nulla nobis at explicabo dignissimos
      commodi mollitia provident quos cupiditate aspernatur.
      Sit quibusdam, porro quis atque voluptatem esse sed explicabo debitis.
      Earum reprehenderit non sit a repellat incidunt nemo odit vel.
    </p>
  </div>
);

export default Home;
