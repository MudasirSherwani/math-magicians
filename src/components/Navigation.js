import React from 'react';
import { Link } from 'react-router-dom';
import './Navigation.css';

const Navigation = () => (
  <div className="container">
    <h1>Math Magicians</h1>
    <div className="navbar">
      <div className="navitem">
        <Link className="item" to="/">
          Home
        </Link>
        <span className="item">|</span>
        <Link className="item" to="/Calculator">
          Calculator
        </Link>
        <span className="item">|</span>
        <Link className="item" to="/Quote">
          Quote
        </Link>
      </div>
    </div>
  </div>
);

export default Navigation;
