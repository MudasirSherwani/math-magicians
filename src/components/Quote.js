import React from 'react';
import './quote.css';

const Quote = () => (
  <div className="quote">
    <div className="quote-text">
      If you want to serve humanity, Your wealth should be in your hands
      <br />
      not in your heart and minds. –M.R Sherwani–
    </div>
  </div>
);

export default Quote;
