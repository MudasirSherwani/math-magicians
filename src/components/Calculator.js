import React, { Component } from 'react';
import calculate from '../logic/calculate';
import './Calculator.css';

export default class Calculator extends Component {
  constructor(props) {
    super(props);
    this.state = {
      total: null,
      next: null,
      operation: null,
    };
  }

  EventHandler = (element) => {
    const btnValue = element.target.textContent;
    const getCalcu = calculate(this.state, btnValue);
    this.setState(getCalcu);
  };

  render() {
    const { total, operation, next } = this.state;
    return (
      <div className="calculator">
        <div className="display">
          <div className="display-value">
            {total}
            {operation}
            {next}
          </div>
        </div>
        <div className="inputs">
          <button type="submit" onClick={this.EventHandler} className="input-btn" value="number">AC</button>
          <button type="submit" onClick={this.EventHandler} className="input-btn" value="number">+/-</button>
          <button type="submit" onClick={this.EventHandler} className="input-btn" value="number">%</button>
          <button type="submit" onClick={this.EventHandler} className="input-btn btn-org" value="number">รท</button>
          <button type="submit" onClick={this.EventHandler} className="input-btn" value="number">7</button>
          <button type="submit" onClick={this.EventHandler} className="input-btn" value="number">8</button>
          <button type="submit" onClick={this.EventHandler} className="input-btn" value="number">9</button>
          <button type="submit" onClick={this.EventHandler} className="input-btn btn-org" value="number">x</button>
          <button type="submit" onClick={this.EventHandler} className="input-btn" value="number">4</button>
          <button type="submit" onClick={this.EventHandler} className="input-btn" value="number">5</button>
          <button type="submit" onClick={this.EventHandler} className="input-btn" value="number">6</button>
          <button type="submit" onClick={this.EventHandler} className="input-btn btn-org" value="number">-</button>
          <button type="submit" onClick={this.EventHandler} className="input-btn" value="number">1</button>
          <button type="submit" onClick={this.EventHandler} className="input-btn" value="number">2</button>
          <button type="submit" onClick={this.EventHandler} className="input-btn" value="number">3</button>
          <button type="submit" onClick={this.EventHandler} className="input-btn btn-org" value="number">+</button>
          <button type="submit" onClick={this.EventHandler} className="input-btn btn-zero" value="number">0</button>
          <button type="submit" onClick={this.EventHandler} className="input-btn" value="number">.</button>
          <button type="submit" onClick={this.EventHandler} className="input-btn btn-org" value="number">=</button>
        </div>
      </div>
    );
  }
}
