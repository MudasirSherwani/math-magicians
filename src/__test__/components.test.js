import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import { BrowserRouter as Router } from 'react-router-dom';
import Navigation from '../components/Navigation';
import Home from '../components/Home';
import Quote from '../components/Quote';
import Calculator from '../components/Calculator';

describe('Navigation renders correctly', () => {
  test('Renders Navigation Correctly', () => {
    const navigation = render(
      <Router>
        <Navigation />
      </Router>,
    );
    expect(navigation).toMatchSnapshot();
  });
});

describe('Home renders correctly', () => {
  test('Renders Header Correctly', () => {
    const home = render(
      <Router>
        <Home />
      </Router>,
    );
    expect(home).toMatchSnapshot();
  });
});

describe('Quote renders correctly', () => {
  test('Renders Header Correctly', () => {
    const quote = render(
      <Router>
        <Quote />
      </Router>,
    );
    expect(quote).toMatchSnapshot();
  });
});

describe('Calculator renders correctly', () => {
  test('Renders Calculator Correctly', () => {
    const calc = render(
      <Router>
        <Calculator />
      </Router>,
    );
    expect(calc).toMatchSnapshot();
  });
});
