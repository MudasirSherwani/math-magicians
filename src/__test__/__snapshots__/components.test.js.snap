// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renders correctly Renders Button Correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <button
        class="input-btn undefined"
        type="button"
      >
        AC
      </button>
      <button
        class="input-btn undefined"
        type="button"
      >
        +/-
      </button>
      <button
        class="input-btn undefined"
        type="button"
      >
        %
      </button>
      <button
        class="input-btn btn-org"
        type="button"
      >
        รท
      </button>
      <button
        class="input-btn undefined"
        type="button"
      >
        7
      </button>
      <button
        class="input-btn undefined"
        type="button"
      >
        8
      </button>
      <button
        class="input-btn undefined"
        type="button"
      >
        9
      </button>
      <button
        class="input-btn btn-org"
        type="button"
      >
        x
      </button>
      <button
        class="input-btn undefined"
        type="button"
      >
        4
      </button>
      <button
        class="input-btn undefined"
        type="button"
      >
        5
      </button>
      <button
        class="input-btn undefined"
        type="button"
      >
        6
      </button>
      <button
        class="input-btn btn-org"
        type="button"
      >
        -
      </button>
      <button
        class="input-btn undefined"
        type="button"
      >
        1
      </button>
      <button
        class="input-btn undefined"
        type="button"
      >
        2
      </button>
      <button
        class="input-btn undefined"
        type="button"
      >
        3
      </button>
      <button
        class="input-btn btn-org"
        type="button"
      >
        +
      </button>
      <button
        class="input-btn btn-zero"
        type="button"
      >
        0
      </button>
      <button
        class="input-btn undefined"
        type="button"
      >
        .
      </button>
      <button
        class="input-btn btn-org"
        type="button"
      >
        =
      </button>
    </div>
  </body>,
  "container": <div>
    <button
      class="input-btn undefined"
      type="button"
    >
      AC
    </button>
    <button
      class="input-btn undefined"
      type="button"
    >
      +/-
    </button>
    <button
      class="input-btn undefined"
      type="button"
    >
      %
    </button>
    <button
      class="input-btn btn-org"
      type="button"
    >
      รท
    </button>
    <button
      class="input-btn undefined"
      type="button"
    >
      7
    </button>
    <button
      class="input-btn undefined"
      type="button"
    >
      8
    </button>
    <button
      class="input-btn undefined"
      type="button"
    >
      9
    </button>
    <button
      class="input-btn btn-org"
      type="button"
    >
      x
    </button>
    <button
      class="input-btn undefined"
      type="button"
    >
      4
    </button>
    <button
      class="input-btn undefined"
      type="button"
    >
      5
    </button>
    <button
      class="input-btn undefined"
      type="button"
    >
      6
    </button>
    <button
      class="input-btn btn-org"
      type="button"
    >
      -
    </button>
    <button
      class="input-btn undefined"
      type="button"
    >
      1
    </button>
    <button
      class="input-btn undefined"
      type="button"
    >
      2
    </button>
    <button
      class="input-btn undefined"
      type="button"
    >
      3
    </button>
    <button
      class="input-btn btn-org"
      type="button"
    >
      +
    </button>
    <button
      class="input-btn btn-zero"
      type="button"
    >
      0
    </button>
    <button
      class="input-btn undefined"
      type="button"
    >
      .
    </button>
    <button
      class="input-btn btn-org"
      type="button"
    >
      =
    </button>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Calculator renders correctly Renders Calculator Correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="calculator-page"
      >
        <div
          class="heading"
        >
          <h2>
            Let's do some math!
          </h2>
        </div>
        <div
          class="calculator"
        >
          <div
            class="display"
          >
            <div
              class="display-value"
            />
          </div>
          <div
            class="inputs"
          >
            <button
              class="input-btn undefined"
              type="button"
            >
              AC
            </button>
            <button
              class="input-btn undefined"
              type="button"
            >
              +/-
            </button>
            <button
              class="input-btn undefined"
              type="button"
            >
              %
            </button>
            <button
              class="input-btn btn-org"
              type="button"
            >
              รท
            </button>
            <button
              class="input-btn undefined"
              type="button"
            >
              7
            </button>
            <button
              class="input-btn undefined"
              type="button"
            >
              8
            </button>
            <button
              class="input-btn undefined"
              type="button"
            >
              9
            </button>
            <button
              class="input-btn btn-org"
              type="button"
            >
              x
            </button>
            <button
              class="input-btn undefined"
              type="button"
            >
              4
            </button>
            <button
              class="input-btn undefined"
              type="button"
            >
              5
            </button>
            <button
              class="input-btn undefined"
              type="button"
            >
              6
            </button>
            <button
              class="input-btn btn-org"
              type="button"
            >
              -
            </button>
            <button
              class="input-btn undefined"
              type="button"
            >
              1
            </button>
            <button
              class="input-btn undefined"
              type="button"
            >
              2
            </button>
            <button
              class="input-btn undefined"
              type="button"
            >
              3
            </button>
            <button
              class="input-btn btn-org"
              type="button"
            >
              +
            </button>
            <button
              class="input-btn btn-zero"
              type="button"
            >
              0
            </button>
            <button
              class="input-btn undefined"
              type="button"
            >
              .
            </button>
            <button
              class="input-btn btn-org"
              type="button"
            >
              =
            </button>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="calculator-page"
    >
      <div
        class="heading"
      >
        <h2>
          Let's do some math!
        </h2>
      </div>
      <div
        class="calculator"
      >
        <div
          class="display"
        >
          <div
            class="display-value"
          />
        </div>
        <div
          class="inputs"
        >
          <button
            class="input-btn undefined"
            type="button"
          >
            AC
          </button>
          <button
            class="input-btn undefined"
            type="button"
          >
            +/-
          </button>
          <button
            class="input-btn undefined"
            type="button"
          >
            %
          </button>
          <button
            class="input-btn btn-org"
            type="button"
          >
            รท
          </button>
          <button
            class="input-btn undefined"
            type="button"
          >
            7
          </button>
          <button
            class="input-btn undefined"
            type="button"
          >
            8
          </button>
          <button
            class="input-btn undefined"
            type="button"
          >
            9
          </button>
          <button
            class="input-btn btn-org"
            type="button"
          >
            x
          </button>
          <button
            class="input-btn undefined"
            type="button"
          >
            4
          </button>
          <button
            class="input-btn undefined"
            type="button"
          >
            5
          </button>
          <button
            class="input-btn undefined"
            type="button"
          >
            6
          </button>
          <button
            class="input-btn btn-org"
            type="button"
          >
            -
          </button>
          <button
            class="input-btn undefined"
            type="button"
          >
            1
          </button>
          <button
            class="input-btn undefined"
            type="button"
          >
            2
          </button>
          <button
            class="input-btn undefined"
            type="button"
          >
            3
          </button>
          <button
            class="input-btn btn-org"
            type="button"
          >
            +
          </button>
          <button
            class="input-btn btn-zero"
            type="button"
          >
            0
          </button>
          <button
            class="input-btn undefined"
            type="button"
          >
            .
          </button>
          <button
            class="input-btn btn-org"
            type="button"
          >
            =
          </button>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Home renders correctly Renders Header Correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="homepage"
      >
        <h2
          class="heading"
        >
          Welcome to our page!
        </h2>
        <p
          class="details"
        >
          Lorem ipsum dolor sit amet consectetur adipisicing elit. Fugit ex fugiat voluptatibus accusamus possimus perspiciatis magni non similique odio debitis, deleniti reprehenderit quae beatae nulla nobis at explicabo dignissimos commodi mollitia provident quos cupiditate aspernatur. Sit quibusdam, porro quis atque voluptatem esse sed explicabo debitis. Earum reprehenderit non sit a repellat incidunt nemo odit vel.
        </p>
        <p
          class="details"
        >
          Lorem ipsum dolor sit amet consectetur adipisicing elit. Fugit ex fugiat voluptatibus accusamus possimus perspiciatis magni non similique odio debitis, deleniti reprehenderit quae beatae nulla nobis at explicabo dignissimos commodi mollitia provident quos cupiditate aspernatur. Sit quibusdam, porro quis atque voluptatem esse sed explicabo debitis. Earum reprehenderit non sit a repellat incidunt nemo odit vel.
        </p>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="homepage"
    >
      <h2
        class="heading"
      >
        Welcome to our page!
      </h2>
      <p
        class="details"
      >
        Lorem ipsum dolor sit amet consectetur adipisicing elit. Fugit ex fugiat voluptatibus accusamus possimus perspiciatis magni non similique odio debitis, deleniti reprehenderit quae beatae nulla nobis at explicabo dignissimos commodi mollitia provident quos cupiditate aspernatur. Sit quibusdam, porro quis atque voluptatem esse sed explicabo debitis. Earum reprehenderit non sit a repellat incidunt nemo odit vel.
      </p>
      <p
        class="details"
      >
        Lorem ipsum dolor sit amet consectetur adipisicing elit. Fugit ex fugiat voluptatibus accusamus possimus perspiciatis magni non similique odio debitis, deleniti reprehenderit quae beatae nulla nobis at explicabo dignissimos commodi mollitia provident quos cupiditate aspernatur. Sit quibusdam, porro quis atque voluptatem esse sed explicabo debitis. Earum reprehenderit non sit a repellat incidunt nemo odit vel.
      </p>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Navigation renders correctly Renders Navigation Correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="container"
        data-testid="Navigation"
      >
        <h1>
          Math Magicians
        </h1>
        <div
          class="navbar"
        >
          <div
            class="navitem"
          >
            <a
              class="item"
              href="/"
            >
              Home
            </a>
            <span
              class="item"
            >
              |
            </span>
            <a
              class="item"
              href="/Calculator"
            >
              Calculator
            </a>
            <span
              class="item"
            >
              |
            </span>
            <a
              class="item"
              href="/Quote"
            >
              Quote
            </a>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="container"
      data-testid="Navigation"
    >
      <h1>
        Math Magicians
      </h1>
      <div
        class="navbar"
      >
        <div
          class="navitem"
        >
          <a
            class="item"
            href="/"
          >
            Home
          </a>
          <span
            class="item"
          >
            |
          </span>
          <a
            class="item"
            href="/Calculator"
          >
            Calculator
          </a>
          <span
            class="item"
          >
            |
          </span>
          <a
            class="item"
            href="/Quote"
          >
            Quote
          </a>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Quote renders correctly Renders Header Correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="quote"
      >
        <div
          class="quote-text"
        >
          If you want to serve humanity, Your wealth should be in your hands
          <br />
          not in your heart and minds. โM.R Sherwaniโ
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="quote"
    >
      <div
        class="quote-text"
      >
        If you want to serve humanity, Your wealth should be in your hands
        <br />
        not in your heart and minds. โM.R Sherwaniโ
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
